scripts/compare_bencher_ab.py

```diff
@@ -0,0 +1,249 @@
+    print(f"{title:^{total_width}}")
+    print("=" * total_width)
+    print()
+
+    # Column widths
+    metric_width = 40
+    bar_width = 50  # Width for the diverging bar chart
+    values_width = 30
+
+    # Print column headers
+    print(
+        f"{'Metric':<{metric_width}} {'Performance Change':^{bar_width}} {'Values':^{values_width}}"
+    )
+    print(
+        f"{'':<{metric_width}} {'← Better  |  Worse →':^{bar_width}} {label1 + ' → ' + label2:^{values_width}}"
+    )
+    print("-" * total_width)
+
+    # Process each metric
+    for key in all_keys:
+        val1 = metrics1.get(key)
+        val2 = metrics2.get(key)
+
+        # Format metric name
+        metric_name = key[: metric_width - 1] if len(key) >= metric_width else key
+
+        # Calculate change
+        change = calculate_percentage_change(val1, val2)
+
+        # Create diverging bar chart with context-aware direction
+        bar_display = create_diverging_bar(change, key, 40)
+
+        # Format values
+        val1_str = f"{val1:.2f}" if val1 is not None else "N/A"
+        val2_str = f"{val2:.2f}" if val2 is not None else "N/A"
+        values_display = f"{val1_str} → {val2_str}"
+
+        # Print row
+        print(
+            f"{metric_name:<{metric_width}} {bar_display:<{bar_width}} {values_display:<{values_width}}"
+        )
+
+    print()
+    print("Legend:")
+    print("  ▓▓▓|         = Improvement (left side, better performance)")
+    print("      |███     = Regression (right side, worse performance)")
+    print("      |         = No significant change (<2%)")
+    print("  ▓ = Better performance (lighter blocks)")
+    print("  █ = Worse performance (darker blocks)")
+    print("  Scale: ±50% change uses full bar width")
+    print(
+        "  Note: For performance metrics, lower latency = better, higher throughput = better"
+    )
+
+    # Summary statistics
+    print()
+    print("Summary:")
+    improvements = 0
+    regressions = 0
+    no_change = 0
+
+    for key in all_keys:
+        val1 = metrics1.get(key)
+        val2 = metrics2.get(key)
+        if val1 is not None and val2 is not None and val1 != 0:
+            change_val = ((val2 - val1) / val1) * 100
+            if abs(change_val) < 2:
+                no_change += 1
+            else:
+                # Use is_higher_better() to determine if this is actually an improvement or regression
+                higher_is_better = is_higher_better(key)
+                is_improvement = (change_val > 0 and higher_is_better) or (
+                    change_val < 0 and not higher_is_better
+                )
+                if is_improvement:
+                    improvements += 1
+                else:
+                    regressions += 1
+
+    total_compared = improvements + regressions + no_change
+    print(f"  Total metrics compared: {total_compared}")
+    print(f"  Improvements: {improvements}")
+    print(f"  Regressions: {regressions}")
```

Comment:
The number of regressions and improvements does seem terribly important. Having a lot of small changes is probably noise, but a single large change may be source for concern. This information is much better communicated by the bars in my opinion.